src/entities/course/ui/course-card/course-card.module.scss

```diff
@@ -0,0 +1,112 @@
+.course-card {
+  $accent-color: var(--accent-bg-color);
+
+  @extend %transition-all;
+
+  // cursor: pointer;
```

Comment:
you'v got a bunch of commented styles in that file

Reply:
Fixed, thank you